maplayer.c

```diff
@@ -1604,7 +1604,8 @@ char* msLayerBuildSQLOrderBy(layerObj *layer)
   if( layer->sortBy.nProperties > 0 ) {
     int i;
     for(i=0;i<layer->sortBy.nProperties;i++) {
-      char* escaped = msLayerEscapePropertyName(layer, layer->sortBy.properties[i].item);
+      char* escaped = msSmallMalloc(strlen(layer->sortBy.properties[i].item) + 3);
```

Comment:
Tx Even, you are totally right!
But we need to enclose the sort property by double quotes - without this Oracle throws an error if the Oracle column or the alias aren't upper case.
What do you think about using the original `msLayerEscapePropertyName` and then enclosing the escaped string in double quotes (if it isn't yet enclosed), like the following:

```
     char* escaped = msLayerEscapePropertyName(layer, layer->sortBy.properties[i].item);
     //Enclose property name in double quotes (if it isn't yet) to ensure that mixed case property names are supported
      if (escaped[0] != '"')
      {
        size_t propertyLen = strlen(escaped);
        escaped = msSmallRealloc(escaped, propertyLen + 3);
        memmove(escaped + 1, escaped, propertyLen);
        escaped[0] = '\"';
        escaped[propertyLen+1] = '\"';
        escaped[propertyLen + 2] = 0;
      }
```
I've added a corresponding commit to the branch